projects/archiver/src/notifications/device-notifications.ts

```diff
@@ -33,8 +62,27 @@ export const scheduleAppStoreNotification = (
         sender: 'editions-backend',
     }
 
-    console.log(
-        'issue app store notification payload',
-        issueAppStoreNotification,
-    )
+    console.log('issue app store notification payload', payload)
+
+    const reqBody = {
+        method: 'POST',
+        headers: {
+            Authorization: `Bearer ${apiKey}`,
+            Accept: 'application/json',
+            'Content-Type': 'application/json',
+        },
+        body: JSON.stringify(payload),
+    }
+
+    const rawResponse = await fetch(reqEndpoint, reqBody)
+    const { status, statusText } = rawResponse
+
+    if (status < 200 || status > 299) {
+        const resText = await attempt(rawResponse.text())
+        console.error('error message:', resText)
```

Comment:
log the status here too? better yet use the same message you're throwing in the error below.

Reply:
done